src/routes/resources/resources.routes.ts

```
@@ -191,29 +191,75 @@ export const getUsersResources = createRoute({
+  path: "/resource/upvote/{id}",
+  method: "patch",
+  tags,
+  responses: {
+    [HttpStatusCodes.OK]: jsonContent(
+      z.object({
+        count: z.number(),
+        success: z.boolean().default(false),
+        resourceId: z.string(),
+      }),
+      "The upvote was successful"
+    ),
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)